pkg/key/handler.go

```
@@ -43,7 +69,14 @@
 
 func (h *Handler) ListIdentities(w http.ResponseWriter, r *http.Request) {
 	identityList := storage.GetUniqueIdentities()
-	htmx.RenderListIdentities(w, identityList)
+	accept_header := r.Header["Accept"]
+	if slices.Contains(accept_header, "application/json") {
+		w.Header().Set("Content-Type", "application/json")
+		json.NewEncoder(w).Encode(identityList)
```

golangci-lint

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)